extensions/application-uninstaller/src/index.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  List,
+  ActionPanel,
+  ActionPanelItem,
```

Comment:
```suggestion
  Action,
```
Deprecated, use Action instead